proposals/wac-ucr/index.bs

```diff
@@ -802,6 +802,29 @@ For example, if the data Carol and Oscar saw in the resume was
 background, she wouldn't want them to know that they were only seeing
 a filtered view.
 
+### Minimal Credential Disclosure ### {#uc-minimalcredentials}
+
+To continue with the  [[#uc-whopermitted]] example, Oscar now wants to
```

Comment:
This references #uc-whopermitted, but it seems that it would make sense for this one to reference [[#capabilities-vc]], since those use cases are specifically related to authorization by verifiable credential.